### DISCUSSION TITLE
Implement an archive state for resources in Mautic

### DISCUSSION THREAD

**Discussion by RCheesley** [mautic/mautic#10427] [Requests For Proposals (RFPs)]
## 🤔  Problem outline

There is currently no way to remove older resources in Mautic which are no longer in use without deleting them completely.

This project involves implementing a state - archived - which would enable the marketer to effectively ‘hide’ resources which are no longer being used.

This project is part of the [Resource Management Strategic Initiative](https://mautic.atlassian.net/wiki/spaces/MI/pages/324403254/Resource+Management).

## ✅  Key requirements

- A state of archived for all resources in Mautic
- The ability to transition state between published, unpublished and archived individually and in batch
- The ability to view archived resources (eg with a filter)
- A permissions level to restrict access to the functionality of archiving, and the viewing of archived resources
- The ability to run a report to show all archived resources

## 💰 Funding

This project currently has a $500 budget, half of the budget for the [Resource Management Initiative](https://opencollective.com/mautic/projects/resource-management-initiative).

Want to add to the fund? Make a donation here: [https://opencollective.com/resource-management-initiative/contribute/implement-an-archive-state-for-resources-in-mautic-32208/checkout](https://opencollective.com/resource-management-initiative/contribute/implement-an-archive-state-for-resources-in-mautic-32208/checkout).

[![open collective badge](https://opencollective.com/resource-management-initiative/tiers/implement-an-archive-state-for-resources-in-mautic/badge.svg?label=Implement%20an%20archive%20state%20for%20resources%20in%20Mautic&color=brightgreen)](https://opencollective.com/resource-management-initiative/contribute/implement-an-archive-state-for-resources-in-mautic-32208/checkout)

## ℹ️  Timelines

As this will be a new feature, it should be created against the 4.x branch for consideration in a forthcoming minor release.

## 👉  How to make a proposal

Please review the information in the [Community Handbook](https://contribute.mautic.org/product-team/request-for-proposals-rfp-process#how-to-claim-a-project) regarding the workflow for proposing and working on this project.

**Comment by nabby27** [comment]
First of all, I'm sorry if this is not the place to communicate this 🙏.

I've seen that you accept funds and I think that the platform we have launched could be good for you. It's called [Opire](https://opire.dev), it's a rewards platform for developers where any person or organization can create rewards on the project that uses Opire. If the rewards are created by people external to the GitHub project, the project receives 5% as a thank-you for maintaining an open-source project. Additionally, rewards accumulate per issue, so several people can contribute their bit to encourage an issue to be resolved. What do you think of the idea of using Opire?

**Comment by BrandWorksApp** [comment]
I know this has nothing to do with this thread but I have searched all over to get connected to the Mautic community. I want to build Mautic locally on my windows computer and I want to use docker to host it but I have to confirue the files of the source code of Mautic 5.1.0. I just face the issue of clear guidance of setting up the files properly.
